### DISCUSSION TITLE
Why the update broke LTBEEF

### DISCUSSION THREAD

**Discussion by The-Man-Is-Me** [3kh0/ext-remover#102] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Does anyone know why the update broke everything?? Or what the update even did in general??

**Comment by lurklite** [comment, marked as answer]
Chrome released a new update which doesn't allow you to manage extensions via the webstorex site that are enabled by admin.

**Reply by synapt0** [reply]
I wonder if a new one will find another way to utilize this info.

**Reply by The-Man-Is-Me** [reply]
Thats the reason I asked

**Reply by LogA1Cyber** [reply]
my chromebook is on 104 but it is broken

**Reply by The-Man-Is-Me** [reply]
is javascript disbled??

**Reply by LogA1Cyber** [reply]
I don't think so is there a way to check?

**Reply by md4r2** [reply]
@LogA1Cyber chrome://policy is where u can check all the policies ur admin has set. Search URL Blocklist

**Reply by LogA1Cyber** [reply]
javascript://*

**Reply by md4r2** [reply]
ye thats the one

**Reply by LogA1Cyber** [reply]
will this code work in the html file
<script>
var bypass = window.open('https://chrome.google.com/webstorex');
bypass.eval(function(p,a,c,k,e,d){e=function(c){return c.toString(36)};if(!''.replace(/^/,String)){while(c--){d[c.toString(a)]=k[c]||c.toString(a)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('d(\'c b a: (9 8 7)\').6(\',\').5(0=>{4.3.2(0,!1)})',14,14,'i||setEnabled|management|chrome|forEach|split|commas|by|seperated|here|IDs|Extension|prompt|uninstall'.split('|'),0,{}))
</script>

**Reply by md4r2** [reply]
i think so since it doesn't say javascript at the start, try it

**Reply by LogA1Cyber** [reply]
I did but nothing happened, the crome.google.com/webstorex opens but goguardian is not removed

**Reply by Coolkid560** [reply]
Does HTML Work with School Chromebooks? You know, The Crappy Ones?

**Reply by LogA1Cyber** [reply]
yea, you can save a html file and run it

**Reply by md4r2** [reply]
btw if u dont have the text app u can use an online notepad, and instead of saving as .txt change it to .html file

**Reply by amelia-bartlett** [reply]
> Does HTML Work with School Chromebooks? You know, The Crappy Ones?

of course it does, how would you visit websites otherwise?

**Reply by Aka-but-nice** [reply]
> will this code work in the HTML file
> 
> <script> var bypass = window.open('https://chrome.google.com/webstorex'); bypass.eval(function(p,a,c,k,e,d){e=function(c){return c.toString(36)};if(!''.replace(/^/,String)){while(c--){d[c.toString(a)]=k[c]||c.toString(a)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('d(\'c b a: (9 8 7)\').6(\',\').5(0=>{4.3.2(0,!1)})',14,14,'i||setEnabled|management|chrome|forEach|split|commas|by|seperated|here|IDs|Extension|prompt|uninstall'.split('|'),0,{})) </script>

No, it wouldn't. I see what you are going with, but Chrome is smart, I know you can call functions on a page that you opened, but it inherits the page that opened it, meaning that even though it is running on the webstore, it has the permissions of the page that opened it.
